firmware/thruna/BMS/Src/App/states/App_PreChargeState.c

```diff
@@ -51,10 +51,6 @@ static void PreChargeStateRunOnTick100Hz(struct StateMachine *const state_machin
             &precharge_fault_limit_exceeded);
         const bool external_shutdown_occurred = !App_Airs_IsAirNegativeClosed(airs);
 
-        struct HeartbeatMonitor *hb_monitor = App_BmsWorld_GetHeartbeatMonitor(world);
-        const bool               missing_hb = !App_SharedHeartbeatMonitor_Tick(hb_monitor);
-        App_CanAlerts_BMS_MissingHeartbeatFault_Set(missing_hb);
```

Comment:
My removing the missing hb error we seem to be entering pre-charge for driving when trying to charge is LV is powered up before the 240v charger plug is connected. (is_charge_connected signal not received until 240v plugged in).

Could re-implement missing hb as a condition to not enter pre-charge for driving in init state (not a fault, but would prevent from entering pre-charge for driving conditions if other boards disconnected)

Reply:
talked on Slack, this is OK to leave for a later date. @will-chaba will make a Jira ticket